index.js

```diff
@@ -1,34 +1,35 @@
-import JsonLdParser from '@rdfjs/parser-jsonld'
-import N3Parser from '@rdfjs/parser-n3'
-import NTriplesSerializer from '@rdfjs/serializer-ntriples'
 import SinkMap from '@rdfjs/sink-map'
-import JsonLdSerializer from './lib/CustomJsonLdSerializer.js'
-import RdfXmlParser from './lib/CustomRdfXmlParser.js'
+import {
+  jsonLdParser,
+  n3Parser,
+  rdfXmlParser
+} from './parsers.js'
+import {
+  jsonLdSerializer,
+  nTriplesSerializer
+} from './serializers.js'
+
+const parsers = new SinkMap()
+const serializers = new SinkMap()
 
 const formats = {
-  parsers: new SinkMap(),
-  serializers: new SinkMap()
+  parsers,
+  serializers
 }
 
-formats.parsers.set('application/ld+json', new JsonLdParser())
-formats.parsers.set('application/trig', new N3Parser())
-formats.parsers.set('application/n-quads', new N3Parser())
-formats.parsers.set('application/n-triples', new N3Parser())
-formats.parsers.set('text/n3', new N3Parser())
-formats.parsers.set('text/turtle', new N3Parser())
-formats.parsers.set('application/rdf+xml', new RdfXmlParser())
+formats.parsers.set('application/ld+json', jsonLdParser)
+formats.parsers.set('application/trig', n3Parser)
+formats.parsers.set('application/n-quads', n3Parser)
+formats.parsers.set('application/n-triples', n3Parser)
+formats.parsers.set('text/n3', n3Parser)
+formats.parsers.set('text/turtle', n3Parser)
+formats.parsers.set('application/rdf+xml', rdfXmlParser)
 
-formats.serializers.set('application/ld+json', new JsonLdSerializer())
-formats.serializers.set('application/n-quads', new NTriplesSerializer())
-formats.serializers.set('application/n-triples', new NTriplesSerializer())
-formats.serializers.set('text/n3', new NTriplesSerializer())
-formats.serializers.set('text/turtle', new NTriplesSerializer())
+formats.serializers.set('application/ld+json', jsonLdSerializer)
+formats.serializers.set('application/n-quads', nTriplesSerializer)
+formats.serializers.set('application/n-triples', nTriplesSerializer)
+formats.serializers.set('text/n3', nTriplesSerializer)
+formats.serializers.set('text/turtle', nTriplesSerializer)
 
+export { parsers, serializers }
 export default formats
-export {
-  JsonLdParser,
-  JsonLdSerializer,
-  N3Parser,
-  NTriplesSerializer,
-  RdfXmlParser
-}
```

Comment:
I removed these exports, as well the two `lib/Custom*` modules since the parsers are lazily instantiated